cra-tf-validate-ignore-goals.json

```diff
@@ -0,0 +1,3 @@
+{
+    "scc_goals": []
```

Comment:
you will need to ignore all of the COS goals (not relevant to the root module here, just the example)
As for the VPC ones, we may need to review each one as a team to decide what to do